xml/System.IO.Log/SequenceNumber.xml

```diff
@@ -292,7 +292,7 @@
         <returns>
           <see langword="true" /> if both <see cref="T:System.IO.Log.SequenceNumber" /> instances are not equal, otherwise, <see langword="false" />.</returns>
         <remarks>
-          <format type="text/markdown"><![CDATA[The equivalent method for this operator is <xref:System.IO.Log.SequenceNumber.Compare%28System.IO.Log.SequenceNumber%2CSystem.IO.Log.SequenceNumber%29?displayProperty=fullName>]]></format>
+          <format type="text/markdown"><![CDATA[The equivalent method for this operator is <xref:System.IO.Log.SequenceNumber.CompareTo*></format>
```

Comment:
But the source actually uses CompareTo for this operator:
https://referencesource.microsoft.com/#System.IO.Log/System/IO/Log/SequenceNumber.cs,73